Formula/mapnik.rb

```diff
@@ -38,8 +47,7 @@ class Mapnik < Formula
 
   def install
     ENV.cxx11
-
-    ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"
+    ENV["PYTHON"] = "python3.9"
```

Comment:
Python 3.9 due to Scons 3.0.1 on stable. HEAD uses Scons 4.1.0, but Python 3.10 might need Scons 4.4.0. May consider using Homebrew's `scons`, but will leave to explore in separate PR.